local-helm/environments/local-docker.yaml

```diff
@@ -145,11 +145,13 @@ isapi:
       nginx.ingress.kubernetes.io/ssl-redirect: "true"
     hosts:
       - paths: 
-          - /identity
-          - /.well-known
-          - /Account
-          - /connect
-          - /Error
+          - /identity/health
+          - /identity/css  
+          - /identity/Account
+          - /identity/Consent
+          - /identity/Error
+          - /identity/connect
+          - /identity/.well-known
```

Comment:
missing /identity/nhsuk-frontend.
I thought there was a js componenet part to the NHS controls too?

Reply:
Added missing route.